cmd/serverless/main.go

```diff
@@ -297,16 +297,20 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	wg.Wait()
 
-	// start the proxy if needed
+	// set up invocation processor in the serverless Daemon to be used for the proxy and/or lifecycle API
+	serverlessDaemon.InvocationProcessor = &invocationlifecycle.LifecycleProcessor{
+		ExtraTags:           serverlessDaemon.ExtraTags,
+		MetricChannel:       serverlessDaemon.MetricAgent.GetMetricChannel(),
+		ProcessTrace:        serverlessDaemon.TraceAgent.Get().Process,
```

Comment:
are we sure that `.MetricAgent` and `.TraceAgent` and `.TraceAgent.Get()` won't be nil at this point?